components/notifier/tests/strategies/test_notification_strategy.py

```diff
@@ -5,7 +5,7 @@
 import pathlib
 import unittest
 
-from strategies.changed_status import get_notable_metrics_from_json, status_changed
+from strategies.notification_strategy import NotableEvents
 
 
 class StrategiesTestCase(unittest.TestCase):
```

Comment:
pycharm says there are to many public methods, should this be ignored or should this file be split?

Reply:
The class "StrategiesTestCase" needs to be split in two classes. I'd suggest one class for testing the notifications for changed status and one for notifications for metrics that have the same status for a long period.